app/assets/javascripts/mobile.js

```diff
@@ -30,10 +31,18 @@ $(document).ready(function(){
 
   /* Drawer menu */
   $('#menu_badge').bind("tap click", function(evt){
-    evt.preventDefault();
     $("#app").toggleClass('draw');
   });
 
+  var drawerSlide = new Hammer(document.getElementById('app'));
```

Comment:
Do you really need to add a new library "just" for this feature? I don't know what's the current policy of the project about adding external libraries but I guess less with use them, less we have things to maintain. @jhass ?

Btw, Firefox OS uses edge gestures to swipe between application. Did you test this with a Firefox OS phone (or the simulator, try the 2.1 version) to see how it reacts?